src/professionals/Show.tsx

```diff
@@ -45,6 +48,45 @@ export const ProfessionalShow = () => {
         <NumberField source="english" />
         <TextField source="about" />
         <EmailField source="email" />
+
+        <WithRecord
+          label="Projects"
+          render={(record) =>
+            record.projects.map((project: ProfessionalProjectsProps) => {
```

Comment:
this patter could generate performance issue, could you create a new component for this render? 
this goes agains Single responsibility principle of solid, this articule has a good example how to resolve this https://konstantinlebedev.com/solid-in-react/